include/alpaka/core/RuntimeMacros.hpp

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2022 Andrea Bocci, Mehmet Yusufoglu */
+#pragma once
+
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+#    define ALPAKA_DEVICE_THROW(MSG)                                                                                  \
```

Comment:
The documentation string is missing.
There should be also a note that you can not catch the error like you can do it with exceptions.